sdcm/remote.py

```diff
@@ -445,7 +445,7 @@ def _make_scp_cmd(self, src, dst, connect_timeout=300, alive_interval=300):
             key_option = '-i %s' % os.path.expanduser(self.key_file)
         command = ("scp -r -o StrictHostKeyChecking=no -o BatchMode=yes "
                    "-o ConnectTimeout=%d -o ServerAliveInterval=%d "
-                   "-o UserKnownHostsFile=%s -P %d %s %s '%s'")
+                   "-o UserKnownHostsFile=%s -P %d -vvv %s %s '%s'")
```

Comment:
What's the purpose for adding the option `-vvv` here?
Is it block something without of it?

Reply:
@it was to catch an issue i constantly have with SCP, hence it happens much less with it (-vvv is adding the verbose level 3 of the command).
as it is a preliminary PR, this flag will be removed on a final PR.